src/ChatBot/Commands/HelloWorldBotCommand.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace Redbeed\OpenOverlay\ChatBot\Commands;
+
+use Carbon\Carbon;
+use Redbeed\OpenOverlay\ChatBot\Twitch\ChatMessage;
+
+class  HelloWorldBotCommand extends BotCommand
+{
+    public $command = '!hello-advance';
+
+    public function response(ChatMessage $chatMessage): string
+    {
+        return implode(' ', [
+            'Hello World.. i mean ' . $chatMessage->username,
```

Comment:
Hello World.. I mean ' . $chatMessage->username,
            'It is ' . Carbon::now()->toString() . '... I think',